marklogic-data-hub/src/test/java/com/marklogic/hub/dataservices/ingestion/BulkIngestTest.java

```diff
@@ -32,8 +32,8 @@ public void setupTest() {
     public void testBulkIngest() {
 
         String prefix = "/bulkIngesterTest";
-        String endpointState = "{\"next\":" + 0 + ", \"uriprefix\":\""+prefix+"\"}";
-        String workUnit      = "{\"taskId\":"+1+"}";
+        String endpointState =  "{\"next\":"+0+"}";
```

Comment:
Can this just be "{}", since we don't care about "next"? It's supposed to go away once we shift to the newer Java client, but I think it'd be better to make it "{}" in the meantime.